src/chainparams.cpp

```diff
@@ -155,10 +155,10 @@ class CMainParams : public CChainParams {
         };
 
         chainTxData = ChainTxData{
-            // Data from rpc: getchaintxstats 4096 0000000000000000000f1c54590ee18d15ec70e68c8cd4cfbadb1b4f11697eee
-            /* nTime    */ 1550374134,
-            /* nTxCount */ 383732546,
-            /* dTxRate  */ 3.685496590998308
+            // Data from RPC: getchaintxstats 4096 00000000000000000007b083989b3d5a937c8a6d3b633130b678fb4780643811
```

Comment:
Why are we no longer using the same block for the `getchaintxstats` call?

Reply:
I would also prefer using the same block.